File: 백준/Bronze/2562. 최댓값/최댓값.c
#include <stdio.h>


int main(void) {
	int arr[9];
	int max = arr[0];
	int num = 0;
	for (int i = 0; i < 9; i++) {
		scanf("%d", &arr[i]);
		if (arr[i] < 1 || arr[i] > 99) {
			scanf("%d", &arr[i]);
		}
		if (max < arr[i]) {
			max = arr[i];
			num = i + 1;
		}
	}
	printf("%d\n", max);
	printf("%d", num);
	return 0;
}

File: 백준/Bronze/2562. 최댓값/최댓값.py
arr = []
for i in range(9):
    num = int(input())
    arr.append(num)
max = arr[0]
place = 1
for i in range(1, 9):
    if(arr[i] > max):
        max = arr[i]
        place = i + 1
print('{}\n{}'.format(max, place))